xbmc/pvr/addons/PVRClient.h

```diff
@@ -1106,28 +1106,28 @@ namespace PVR
     static xbmc_codec_t cb_get_codec_by_name(const void* kodiInstance, const char* strCodecName);
     //@}
 
-    std::atomic<bool>      m_bReadyToUse;          /*!< true if this add-on is initialised (ADDON_Create returned true), false otherwise */
-    std::atomic<bool>      m_bBlockAddonCalls;     /*!< true if no add-on API calls are allowed */
-    PVR_CONNECTION_STATE   m_connectionState;      /*!< the backend connection state */
-    PVR_CONNECTION_STATE   m_prevConnectionState;  /*!< the previous backend connection state */
-    bool                   m_ignoreClient;         /*!< signals to PVRManager to ignore this client until it has been connected */
+    std::atomic<bool>      m_bReadyToUse; /*!< true if this add-on is initialised (ADDON_Create returned true), false otherwise */
+    std::atomic<bool>      m_bBlockAddonCalls; /*!< true if no add-on API calls are allowed */
```

Comment:
you forgot some space or tabs ?

Reply:
Did not include tabs, thanks. Will update later.